### PR TITLE
Fix wal-g copy for pg

### PR DESCRIPTION
**Pull request by munakoiso**

### CODE REVIEW

**Review by debebantur — APPROVED** [review]

**Comment by debebantur** [comment]
Can you please change without-history flag [here](https://github.com/wal-g/wal-g/blob/fade5e6d54832e584ea8cff0692672ad50f1f7ae/docker/pg_tests/scripts/tests/copy_backup_test.sh#L43C86-L43C101) and [here](https://github.com/wal-g/wal-g/blob/master/docs/PostgreSQL.md#copy)

File: cmd/pg/copy.go [hunk 1]
```diff
@@ -57,11 +57,11 @@ func init() {
...
 	backupCopyCmd.Flags().StringVarP(&fromConfigFile, fromFlag, fromShorthand, "", fromDescription)
-	backupCopyCmd.Flags().BoolVarP(&withoutHistory,
-		withoutHistoryFlag,
-		withoutHistoryShorthand,
+	backupCopyCmd.Flags().BoolVarP(&withAllHistory,
+		withAllHistoryFlag,
+		withAllHistoryShorthand,
 		false,
-		withoutHistoryDescription)
+		withAllHistoryDescription)
```
Comment (ostinru) [inline cmd/pg/copy.go:64]: it is backward incompatible changes... with changed defaults.
Is there real reason to do so?

Comment (munakoiso) [reply cmd/pg/copy.go:64]: Backup without any WALs will not work anyway. So there is no chance that someone has used this code before (and then was able to restore the backup).

Comment (munakoiso) [reply cmd/pg/copy.go:64]: In addition, there was an error without the "without history" flag - the function copied only from finish_lsn. Therefore, backups from walg copy were useless

**Pull request merged**